pkg/server/testing/fake_images_client.go

```diff
@@ -0,0 +1,170 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/containerd/containerd/api/services/images"
+	google_protobuf "github.com/golang/protobuf/ptypes/empty"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+// FakeImagesClient is a simple fake images client, so that cri-containerd
+// can be run for testing without requiring a real containerd setup.
+type FakeImagesClient struct {
+	sync.Mutex
+	called    []CalledDetail
+	errors    map[string]error
+	ImageList map[string]images.Image
+}
+
+var _ images.ImagesClient = &FakeImagesClient{}
+
+// NewFakeImagesClient creates a FakeImagesClient
+func NewFakeImagesClient() *FakeImagesClient {
+	return &FakeImagesClient{
+		errors:    make(map[string]error),
+		ImageList: make(map[string]images.Image),
+	}
+}
+
+func (f *FakeImagesClient) popError(op string) error {
```

Comment:
The other services are all using `popError`. Let's make them consistent.

We could change them all in a future PR if you don't like the name. :)